source/Plugins/ExpressionParser/Swift/SwiftREPL.cpp

```diff
@@ -43,202 +43,232 @@ using namespace lldb_private;
 lldb::REPLSP SwiftREPL::CreateInstance(Status &err, lldb::LanguageType language,
                                        Debugger *debugger, Target *target,
                                        const char *repl_options) {
-  if (language != eLanguageTypeSwift) {
+  assert(language == eLanguageTypeSwift && "can only create a REPL for Swift");
```

Comment:
This isn't right.  If you asked for a eLanguageTypeC++ REPL, then the REPL Factory would go through all the things that could create REPL's asking if it could create one for C++.  The proper response is to return an empty REPLSP meaning, "no I can't create a REPL for C++".  Then the factory for REPL's would go on to the next one.

Reply:
I'll switch back to the original behavior.

Ideally EnumerateSupportedLanguages would make this unnecessary, and a test would have caught this :/.